modules/ipi-install-retrieving-the-openshift-installer.adoc

```diff
@@ -6,11 +6,21 @@
 [id="retrieving-the-openshift-installer_{context}"]
 = Retrieving the {product-title} installer
 
-Use the `stable-4.x` version of the installer to deploy the generally
+Use the `stable-4.x` version of the installer based on your selected architecture to deploy the generally
 available stable version of {product-title}:
 
+.For x86_64
 [source,terminal,subs="attributes"]
 ----
 $ export VERSION=stable-{product-version}
-export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}')
+$ export RELEASE_ARCH=x86_64
+export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/x86_64/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}')
+----
+
+.For aarch64
+[source,terminal,subs="attributes"]
+----
+$ export VERSION=stable-{product-version}
+$ export RELEASE_ARCH=aarch64
+export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/aarch64/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}')
```

Comment:
https://mirror.openshift.com/pub/openshift-v4/aarch64/ocp/$VERSION/release.txt
should be
https://mirror.openshift.com/pub/openshift-v4/$RELEASE_ARCH/ocp/$VERSION/release.txt